code/game/jobs/job/cargo_service.dm

```diff
@@ -111,6 +111,7 @@ Bartender
 /datum/outfit/job/bartender
 	name = "Bartender"
 
+	glasses = /obj/item/clothing/glasses/sunglasses/reagent
```

Comment:
Is that really a problem, @phil235?

He's still not immune to flashbangs and starts out with a shotgun, anyway. Given that flashes are no longer the dominant crowd control these days (thanks to telebatons and flash changes), this isn't nearly as big of a deal as it was over a year ago.